### ISSUE TITLE
Error fetching URL

### ISSUE THREAD

**Issue by jeremydouglass** [processing/processing-library-template-ant#13]
The library template seems to have a default configuration set that is pointed at a bad URL. Or perhaps I've accidentally left it unconfigured in some way that isn't covered by [Library Basics](https://github.com/processing/processing/wiki/Library-Basics). Everything works, but it generates warning messages.

Here is the warning on Ant compile:

```
[javadoc] Constructing Javadoc information...
[javadoc] Registered Taglet ExampleTaglet ...
[javadoc] Standard Doclet version 1.8.0_51
[javadoc] Building tree for all the packages and classes...
[javadoc] javadoc: warning - Error fetching URL: http://processing.org/reference/javadoc/core/
```

[That url is 404](http://processing.org/reference/javadoc/core/).

**Comment by jeremydouglass** [comment]
...looks like it is line 171 of build.properties.

```
javadoc.processing.href=http://processing.org/reference/javadoc/core/
```

Submitted a pull request.

**Comment by prisonerjohn** [comment]
Fixed with PR #14
